nuxt-app/package.json

```diff
@@ -16,9 +16,7 @@
   "devDependencies": {
     "deepmerge": "4.3.1",
     "nuxt": "3.12.4",
-    "sass": "1.70.0",
-    "vue": "3.4.37",
```

Comment:
We import from both of these packages in `layouts/default.vue`, do those need updating?

Reply:
`nuxt` already has `vue-router@npm:4.4.0`  and `vue@npm:3.4.33` as deps, so it's not quite different in terms of version.